### PR TITLE
Registration page two

### LINKED ISSUE #27: Registration Page 2: Create other registration pages with industry, title, position, company, education, location

### PR DESCRIPTION
**Pull request by mohaimenhasan**
Completed the second page with all values and value tracking. Moved constants to different files

Ignore the third page from this commit 
Closes #27

### CODE REVIEW

File: src/Components/Registration/industry.json [hunk 1]
```diff
@@ -0,0 +1,443 @@
+[
```
Comment (ahmedhamodi) [inline src/Components/Registration/industry.json:1]: Use the shorter list (20 industries from Max mentors thing)

Comment (ahmedhamodi) [reply src/Components/Registration/industry.json:1]: Also add tag field in the UI to allow for other custom fields

File: src/Components/LandingPage/MembershipCard.js [hunk 2]
```diff
@@ -73,7 +71,7 @@ class MembershipCard extends Component {
     return (
       <div>
         <div className={classes.card}>
-          <img className={classes.image} src={this.props.type === 'aspiring_professional' ? AspiringProfessional : SeniorExecutive} alt="Membership Image"/>
+          <img className={classes.image} src={this.props.type === 'aspiring_professional' ? AspiringProfessional : SeniorExecutive} alt="Membership_im"/>
```
[the remark this answers is not in the document]
Comment (mohaimenhasan) [reply src/Components/LandingPage/MembershipCard.js:74]: LMAO turns out if you leave the word as "image" under alt tag then react sends out a warning but I will change it to something else

File: src/Components/Registration/AgeGroups.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+const AgeGroups = [
+    {
+        value: '0-17',
+        label: 'Under 17',
```
[the remark this answers is not in the document]
Comment (mohaimenhasan) [reply src/Components/Registration/AgeGroups.js:4]: Sounds good

File: src/Components/Registration/Provinces.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+const Provinces = [
```
[the remark this answers is not in the document]
Comment (mohaimenhasan) [reply src/Components/Registration/Provinces.js:1]: I will add the option to change to United States

File: src/Components/Registration/ThirdPage.js [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+    }
+
+    handleSave(files) {
+        let document = "";
+        let reader = new FileReader();
+        reader.readAsDataURL(files[0]);
+        reader.onload = function () {
+            // Saving files to state for further use and closing Modal.
+            document = reader.result;
+            console.log(this);
```
[the remark this answers is not in the document]
Comment (mohaimenhasan) [reply src/Components/Registration/ThirdPage.js:130]: I think I added it as a part of this page for debugging purpose. But I will fix this with the other branch that has the third page

File: src/Components/LandingPage/MembershipCard.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (ahmedhamodi) [reply src/Components/LandingPage/MembershipCard.js]: woah I did not know that

**Pull request merged**